atomic_reactor/constants.py

```diff
@@ -86,7 +86,7 @@
 PLUGIN_BUMP_RELEASE_KEY = 'bump_release'
 PLUGIN_DISTGIT_FETCH_KEY = 'distgit_fetch_artefacts'
 PLUGIN_FETCH_MAVEN_KEY = 'fetch_maven_artifacts'
-PLUGIN_GENERATE_MAVEN_METADATA_KEY = 'generate_maven_metadata'
+PLUGIN_URL_SOURCES_METADATA_KEY = 'url_sources_metadata'
```

Comment:
Could we have "maven" in the name? Sorry to request a second rename, but I probably wouldn't notice the two plugins are related otherwise

Reply:
renamed it to `maven_url_sources_metadata`